test/EndToEnd/NuGet.Tests.psm1

```diff
@@ -31,6 +31,10 @@ if (!(Test-Path $nugetExePath))
     $nugetExePath = "$nugetRoot\test\EndToEnd\NuGet.exe"
 }
 
+# Enable NuGet Test Mode
```

Comment:
Now also setting the env var when starting the process from within the tests: https://github.com/NuGet/NuGet.Client/commit/cc756a992fedcdc2d1d3ce9e1232f4aa14d8e409